ev3dev/core.py

```diff
@@ -42,6 +42,7 @@
 import fnmatch
 import numbers
 import array
+import logging
```

Comment:
I wonder how much does this add to startup time.

Reply:
No noticable difference as far as I can tell